yt_dlp/extractor/zaiko.py

```diff
@@ -80,14 +82,21 @@ def _real_extract(self, url):
             'cancelled': ('not_live', 'Event has been cancelled', True),
         }.get(status) or ('not_live', f'Unknown event status "{status}"', False)
 
-        stream_url = traverse_obj(player_meta, ('initial_event_info', 'endpoint', {url_or_none}))
+        if traverse_obj(initial_event_info, ('is_jwt_protected', {bool})):
+            stream_url = self._download_json(traverse_obj(
+                initial_event_info, ('jwt_token_url', {url_or_none})), video_id,
+                note='Downloading jwt-protected stream url',
+                errnote='Failed to download jwt-protected stream url')['playback_url']
```

Comment:
don't need the traversal here if it's otherwise fatal
```suggestion
            stream_url = self._download_json(
                initial_event_info['jwt_token_url'], video_id, 'Downloading JWT-protected stream URL',
                'Failed to download JWT-protected stream URL')['playback_url']
```